policy/controller.go

```diff
@@ -39,17 +39,43 @@ type runParameters map[string]string
 // SetVariations prints the test variations formatted as a multi-line
 // github output parameter. The contents are json formatted.
 func (p runParameters) SetVariations(op io.Writer, tv TestVariations) error {
-	switch p["trigger"] {
-	case "is_pr":
-		tv["conf"] = []string{"sha256"}
-		tv["pump"] = []string{"$ECR/tyk-pump:master"}
-		tv["sink"] = []string{"$ECR/tyk-sink:master"}
-	case "is_tag":
-	// Defaults are fine
-	case "is_lts":
-		tv["conf"] = []string{"sha256"}
-		tv["pump"] = []string{"tykio/tyk-pump-docker-pub:v1.8"}
-		tv["sink"] = []string{"tykio/tyk-mdcb-docker:v2.4"}
+
+	if p["job"] == "api-test" {
+		switch p["trigger"] {
+		case "is_pr":
+			tv["api_conf"] = []string{"sha256"}
```

Comment:
Instead of an if inside a case, use `tv[p["job"]+"conf"]` here. The should eliminate the need for a branch.

Reply:
I understand the point of removing the IF, but I think this defeats the purpose of having job as input if we remove the IF.  Then how are we going to set different variations based on input job? we'll need to add an IF at some point for instance if we want to run UI on some specific DBs. Maybe is too early to talk about this since there;'s no real difference now between the 2 jobs